### ISSUE TITLE
Nested objects and arrays does not work after last security patch

### ISSUE THREAD

**Issue by ABuffSeagull** [edmundhung/conform#603]
### Describe the bug and the expected behavior

With a nested schema such as
```ts
type Schema = {
  outside: Array<{ inside: Array<string> }>
}
```
the insert intent does not work. It silently fails and does not add anything.

### Conform version

v1.1.1 and higher

### Steps to Reproduce the Bug or Issue

Repro here: https://github.com/ABuffSeagull/conform-nested-bug

Clicking on the "Add Outside" works fine, but the "Add Inside" does not work. 

### What browsers are you seeing the problem on?

Chrome, Firefox

**Comment by edmundhung** [comment]
Thanks for reporting the issue! This is fixed in [v1.1.3](https://github.com/edmundhung/conform/releases/tag/v1.1.3).

**Comment by ABuffSeagull** [comment]
Perfect, thank you so much! Love this library
